src/db/group_invitations.js

```diff
@@ -0,0 +1,42 @@
+const database = require('../database')
+
+const GroupInvitation = class {
+  constructor (invite) {
+    this._id = invite.id
+    this.userId = invite.user_id
+    this.groupId = invite.group_id
+    this.token = invite.token
+  }
+
+  createLink (host) {
+    return host + '/join_group?token=' + this.token
```

Comment:
Consider usung a shorter invite-url, i.e. `$HOST/invite/1234`

Reply:
Using `$HOST/invite/1234` as a url breakes the html creator, it will assign incorrect js paths. So the query option should be used (for now)